### PR TITLE
[ODS-4237] Ed-Fi-Alliance repo paths in nuspec files

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

File: Application/EdFi.Ods.SandboxAdmin.Web/EdFi.Ods.SandboxAdmin.Web.csproj [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
   <PropertyGroup>
+	<PackageLicenseFile>LICENSE.txt</PackageLicenseFile>
```
Comment (jamessmckay) [inline Application/EdFi.Ods.SandboxAdmin.Web/EdFi.Ods.SandboxAdmin.Web.csproj:5]: fix the formatting here please

File: Application/EdFi.Ods.SandboxAdmin.Web/EdFi.Ods.SandboxAdmin.Web.csproj [hunk 2]
```diff
@@ -292,6 +293,9 @@
       <Name>EdFi.Ods.Common</Name>
     </ProjectReference>
   </ItemGroup>
+ <ItemGroup>
```
Comment (jamessmckay) [inline Application/EdFi.Ods.SandboxAdmin.Web/EdFi.Ods.SandboxAdmin.Web.csproj:296]: fix the formatting please

Comment (jamessmckay) [reply Application/EdFi.Ods.SandboxAdmin.Web/EdFi.Ods.SandboxAdmin.Web.csproj:296]: In fact you can include it into the item group below

File: Application/EdFi.Ods.SandboxAdmin.Web/EdFi.Ods.SandboxAdmin.Web.nuspec [hunk 3]
```diff
@@ -5,8 +5,13 @@
     <version>0.0.0</version>
     <authors>Ed-Fi Alliance</authors>
     <owners>Ed-Fi Alliance</owners>
+	<projectUrl>https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation</projectUrl>
```
Comment (jamessmckay) [inline Application/EdFi.Ods.SandboxAdmin.Web/EdFi.Ods.SandboxAdmin.Web.nuspec:8]: fix formatting

File: Application/EdFi.Ods.SandboxAdmin.Web/EdFi.Ods.SandboxAdmin.Web.nuspec [hunk 4]
```diff
@@ -5,8 +5,13 @@
...
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <description>EdFi.Ods.SandboxAdmin.Web</description>
     <releaseNotes></releaseNotes>
+	<license type="file">LICENSE.txt</license>
```
Comment (jamessmckay) [inline Application/EdFi.Ods.SandboxAdmin.Web/EdFi.Ods.SandboxAdmin.Web.nuspec:12]: fix formatting please

File: Application/EdFi.Ods.SwaggerUI/EdFi.Ods.SwaggerUI.csproj [hunk 5]
```diff
@@ -51,6 +51,12 @@
     <DeployOnBuild Condition=" '$(BuildPackages)'!='' ">$(BuildPackages)</DeployOnBuild>
     <DesktopBuildPackageLocation Condition=" '$(BuildPackages)'!='' ">$(DeployPackageLocation)\$(AssemblyName).zip</DesktopBuildPackageLocation>
   </PropertyGroup>
+  <PropertyGroup>
```
Comment (jamessmckay) [inline Application/EdFi.Ods.SwaggerUI/EdFi.Ods.SwaggerUI.csproj:54]: fix formatting

File: Application/EdFi.Ods.SwaggerUI/EdFi.Ods.SwaggerUI.csproj [hunk 6]
```diff
@@ -51,6 +51,12 @@
...
+        <PackageLicenseFile>LICENSE.txt</PackageLicenseFile>
+   </PropertyGroup>
+   <ItemGroup>
```
Comment (jamessmckay) [inline Application/EdFi.Ods.SwaggerUI/EdFi.Ods.SwaggerUI.csproj:57]: fix formatting

File: Application/EdFi.Ods.SwaggerUI/EdFi.Ods.SwaggerUI.nuspec [hunk 7]
```diff
@@ -5,8 +5,13 @@
     <version>0.0.0</version>
     <authors>Ed-Fi Alliance</authors>
     <owners>Ed-Fi Alliance</owners>
+	<projectUrl>https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation</projectUrl>
```
Comment (jamessmckay) [inline Application/EdFi.Ods.SwaggerUI/EdFi.Ods.SwaggerUI.nuspec:8]: fix formatting

File: Application/EdFi.Ods.SwaggerUI/EdFi.Ods.SwaggerUI.nuspec [hunk 8]
```diff
@@ -5,8 +5,13 @@
...
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <description>EdFi.Ods.SwaggerUI</description>
     <releaseNotes></releaseNotes>
+	<license type="file">LICENSE.txt</license>
```
Comment (jamessmckay) [inline Application/EdFi.Ods.SwaggerUI/EdFi.Ods.SwaggerUI.nuspec:12]: fix formatting

File: Application/EdFi.Ods.WebApi/EdFi.Ods.WebApi.csproj [hunk 9]
```diff
@@ -280,7 +281,7 @@
     <Compile Include="Startup\AzureCloudOdsStartup.cs" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\..\Ed-Fi-ODS\Application\EdFi.Common\EdFi.Common.csproj">
+      <ProjectReference Include="..\..\..\Ed-Fi-ODS\Application\EdFi.Common\EdFi.Common.csproj">
```
Comment (jamessmckay) [inline Application/EdFi.Ods.WebApi/EdFi.Ods.WebApi.csproj:284]: remove xtra white space and fix formatting

File: Application/EdFi.Ods.WebApi/EdFi.Ods.WebApi.csproj [hunk 10]
```diff
@@ -321,6 +322,9 @@
       <Name>EdFi.Ods.Extensions.TPDM</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
```
Comment (jamessmckay) [inline Application/EdFi.Ods.WebApi/EdFi.Ods.WebApi.csproj:325]: fix formatting

File: Application/EdFi.Ods.WebApi/EdFi.Ods.WebApi.nuspec [hunk 11]
```diff
@@ -5,8 +5,13 @@
     <version>0.0.0</version>
     <authors>Ed-Fi Alliance</authors>
     <owners>Ed-Fi Alliance</owners>
+	<projectUrl>https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation</projectUrl>
```
Comment (jamessmckay) [inline Application/EdFi.Ods.WebApi/EdFi.Ods.WebApi.nuspec:8]: fix formatting

File: Application/EdFi.Ods.WebApi/EdFi.Ods.WebApi.nuspec [hunk 12]
```diff
@@ -5,8 +5,13 @@
...
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <description>EdFi.Ods.WebApi</description>
     <releaseNotes></releaseNotes>
+	<license type="file">LICENSE.txt</license>
```
Comment (jamessmckay) [inline Application/EdFi.Ods.WebApi/EdFi.Ods.WebApi.nuspec:12]: fix formatting

File: Scripts/NuGet/EdFi.Installer.SandboxAdmin/EdFi.Installer.SandboxAdmin.nuspec [hunk 13]
```diff
@@ -5,16 +5,18 @@
     <version>1.0.0</version>
     <title>Ed-Fi SwaggerUI Installer Scripts</title>
     <authors>Ed-Fi Alliance</authors>
-    <projectUrl>https://github.com/Ed-Fi-Alliance/Ed-Fi-ODS-Implementation</projectUrl>
+    <projectUrl>https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation</projectUrl>
     <copyright>Copyright © 2020, Ed-Fi Alliance, LLC and contributors.</copyright>
     <summary>Application installer modules for the Ed-Fi ODS/API Sandbox Admin application</summary>
     <description>Application installer modules for the Ed-Fi ODS/API Sandbox Admin application</description>
+	<license type="file">LICENSE.txt</license>
```
Comment (jamessmckay) [inline Scripts/NuGet/EdFi.Installer.SandboxAdmin/EdFi.Installer.SandboxAdmin.nuspec:12]: fix formatting

File: Scripts/NuGet/EdFi.Installer.SandboxAdmin/EdFi.Installer.SandboxAdmin.nuspec [hunk 14]
```diff
@@ -5,16 +5,18 @@
...
   </metadata>
   <files>  
     <file src="Install-EdFiOdsSandboxAdmin.psm1" />
     <file src="initialization.config" />
     <file src="Ed-Fi-Common/**" />
     <file src="Ed-Fi-ODS/**" />
     <file src="Ed-Fi-ODS-Implementation/**" />
+	<file src="../../../LICENSE.txt" />
```
Comment (jamessmckay) [inline Scripts/NuGet/EdFi.Installer.SandboxAdmin/EdFi.Installer.SandboxAdmin.nuspec:20]: fix formatting

File: Scripts/NuGet/EdFi.Installer.SwaggerUI/EdFi.Installer.SwaggerUI.nuspec [hunk 15]
```diff
@@ -5,15 +5,17 @@
...
     <summary>Application installer modules for the Ed-Fi SwaggerUI application</summary>
     <description>Application installer modules for the Ed-Fi SwaggerUI application</description>
+	<license type="file">LICENSE.txt</license>
   </metadata>
   <files>  
     <file src="Install-EdFiOdsSwaggerUI.psm1" target="" />
     <file src="Ed-Fi-Common/**" />
     <file src="Ed-Fi-ODS/**" />
     <file src="Ed-Fi-ODS-Implementation/**" />
+	<file src="../../../LICENSE.txt" />
```
Comment (jamessmckay) [inline Scripts/NuGet/EdFi.Installer.SwaggerUI/EdFi.Installer.SwaggerUI.nuspec:19]: fix formatting

File: Scripts/NuGet/EdFi.Installer.SwaggerUI/EdFi.Installer.SwaggerUI.nuspec [hunk 16]
```diff
@@ -5,15 +5,17 @@
     <version>1.0.0</version>
     <title>Ed-Fi SwaggerUI Installer Scripts</title>
     <authors>Ed-Fi Alliance</authors>
-    <projectUrl>https://github.com/Ed-Fi-Alliance/Ed-Fi-ODS-Implementation</projectUrl>
+    <projectUrl>https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation</projectUrl>
     <copyright>Copyright © 2020, Ed-Fi Alliance, LLC and contributors.</copyright>
     <summary>Application installer modules for the Ed-Fi SwaggerUI application</summary>
     <description>Application installer modules for the Ed-Fi SwaggerUI application</description>
+	<license type="file">LICENSE.txt</license>
```
Comment (jamessmckay) [inline Scripts/NuGet/EdFi.Installer.SwaggerUI/EdFi.Installer.SwaggerUI.nuspec:12]: fix formatting

File: Scripts/NuGet/EdFi.Installer.WebApi/EdFi.Installer.WebApi.nuspec [hunk 17]
```diff
@@ -5,15 +5,17 @@
     <version>1.0.0</version>
     <title>ODS/API WebApi Installer Scripts</title>
     <authors>Ed-Fi Alliance</authors>
-    <projectUrl>https://github.com/Ed-Fi-Alliance/Ed-Fi-ODS-Implementation</projectUrl>
+    <projectUrl>https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation</projectUrl>
     <copyright>Copyright © 2020, Ed-Fi Alliance, LLC and contributors.</copyright>
     <summary>Application installer modules for the Ed-Fi ODS/API WebApi application</summary>
     <description>Application installer modules for the Ed-Fi ODS/API WebApi application</description>
+	<license type="file">LICENSE.txt</license>
```
Comment (jamessmckay) [inline Scripts/NuGet/EdFi.Installer.WebApi/EdFi.Installer.WebApi.nuspec:12]: fix formatting

File: Scripts/NuGet/EdFi.Installer.WebApi/EdFi.Installer.WebApi.nuspec [hunk 18]
```diff
@@ -5,15 +5,17 @@
...
   </metadata>
   <files>  
     <file src="Install-EdFiOdsWebApi.psm1" />
     <file src="Ed-Fi-Common/**" />
     <file src="Ed-Fi-ODS/**" />
     <file src="Ed-Fi-ODS-Implementation/**" />
+	<file src="../../../LICENSE.txt" />
```
Comment (jamessmckay) [inline Scripts/NuGet/EdFi.Installer.WebApi/EdFi.Installer.WebApi.nuspec:19]: fix formatting

File: Application/EdFi.Ods.SandboxAdmin.Web/EdFi.Ods.SandboxAdmin.Web.csproj (hunk 1 above)
Comment (semalaiappan) [reply Application/EdFi.Ods.SandboxAdmin.Web/EdFi.Ods.SandboxAdmin.Web.csproj]: Fixed it . Thanks

File: Application/EdFi.Ods.SandboxAdmin.Web/EdFi.Ods.SandboxAdmin.Web.csproj (hunk 2 above)
Comment (semalaiappan) [reply Application/EdFi.Ods.SandboxAdmin.Web/EdFi.Ods.SandboxAdmin.Web.csproj]: Fixed it . Thanks

File: Application/EdFi.Ods.SandboxAdmin.Web/EdFi.Ods.SandboxAdmin.Web.nuspec (hunk 3 above)
Comment (semalaiappan) [reply Application/EdFi.Ods.SandboxAdmin.Web/EdFi.Ods.SandboxAdmin.Web.nuspec]: Fixed it . Thanks

File: Application/EdFi.Ods.SandboxAdmin.Web/EdFi.Ods.SandboxAdmin.Web.nuspec (hunk 4 above)
Comment (semalaiappan) [reply Application/EdFi.Ods.SandboxAdmin.Web/EdFi.Ods.SandboxAdmin.Web.nuspec]: Fixed it . Thanks

File: Application/EdFi.Ods.SwaggerUI/EdFi.Ods.SwaggerUI.csproj (hunk 5 above)
Comment (semalaiappan) [reply Application/EdFi.Ods.SwaggerUI/EdFi.Ods.SwaggerUI.csproj:54]: Fixed it . Thanks

File: Application/EdFi.Ods.WebApi/EdFi.Ods.WebApi.csproj (hunk 10 above)
Comment (semalaiappan) [reply Application/EdFi.Ods.WebApi/EdFi.Ods.WebApi.csproj:325]: Fixed it  .Thanks

File: Scripts/NuGet/EdFi.Installer.SandboxAdmin/EdFi.Installer.SandboxAdmin.nuspec (hunk 14 above)
Comment (semalaiappan) [reply Scripts/NuGet/EdFi.Installer.SandboxAdmin/EdFi.Installer.SandboxAdmin.nuspec]: Fixed it  .Thanks

File: Scripts/NuGet/EdFi.Installer.SwaggerUI/EdFi.Installer.SwaggerUI.nuspec (hunk 15 above)
Comment (semalaiappan) [reply Scripts/NuGet/EdFi.Installer.SwaggerUI/EdFi.Installer.SwaggerUI.nuspec]: Fixed it  .Thanks

File: Scripts/NuGet/EdFi.Installer.SwaggerUI/EdFi.Installer.SwaggerUI.nuspec (hunk 16 above)
Comment (semalaiappan) [reply Scripts/NuGet/EdFi.Installer.SwaggerUI/EdFi.Installer.SwaggerUI.nuspec]: Fixed it  .Thanks

File: Scripts/NuGet/EdFi.Installer.WebApi/EdFi.Installer.WebApi.nuspec (hunk 18 above)
Comment (semalaiappan) [reply Scripts/NuGet/EdFi.Installer.WebApi/EdFi.Installer.WebApi.nuspec]: Fixed it  .Thanks

File: Application/EdFi.Ods.SwaggerUI/EdFi.Ods.SwaggerUI.csproj (hunk 6 above)
Comment (semalaiappan) [reply Application/EdFi.Ods.SwaggerUI/EdFi.Ods.SwaggerUI.csproj]: Fixed it  .Thanks

File: Application/EdFi.Ods.SwaggerUI/EdFi.Ods.SwaggerUI.nuspec (hunk 7 above)
Comment (semalaiappan) [reply Application/EdFi.Ods.SwaggerUI/EdFi.Ods.SwaggerUI.nuspec]: Fixed it  .Thanks

File: Application/EdFi.Ods.SwaggerUI/EdFi.Ods.SwaggerUI.nuspec (hunk 8 above)
Comment (semalaiappan) [reply Application/EdFi.Ods.SwaggerUI/EdFi.Ods.SwaggerUI.nuspec]: Fixed it  .Thanks

File: Application/EdFi.Ods.WebApi/EdFi.Ods.WebApi.csproj (hunk 9 above)
Comment (semalaiappan) [reply Application/EdFi.Ods.WebApi/EdFi.Ods.WebApi.csproj]: Fixed it .Thanks

File: Application/EdFi.Ods.WebApi/EdFi.Ods.WebApi.nuspec (hunk 11 above)
Comment (semalaiappan) [reply Application/EdFi.Ods.WebApi/EdFi.Ods.WebApi.nuspec]: Fixed it .Thanks

File: Scripts/NuGet/EdFi.Installer.SandboxAdmin/EdFi.Installer.SandboxAdmin.nuspec (hunk 13 above)
Comment (semalaiappan) [reply Scripts/NuGet/EdFi.Installer.SandboxAdmin/EdFi.Installer.SandboxAdmin.nuspec]: Fixed it .Thanks

File: Scripts/NuGet/EdFi.Installer.WebApi/EdFi.Installer.WebApi.nuspec (hunk 17 above)
Comment (semalaiappan) [reply Scripts/NuGet/EdFi.Installer.WebApi/EdFi.Installer.WebApi.nuspec]: Fixed it .Thanks

**Comment by semalaiappan** [comment]
I fixed formatting issue for all files . Ready for review again

**Pull request merged**